heareval/tasks/pipeline.py

```diff
@@ -170,6 +170,33 @@ def requires(self):
         # This should have something like the following:
         # return { "train": self.train, "test": self.test }
 
+    def get_requires_metadata(self, requires_key: str) -> pd.DataFrame:
+        """
+        For a particular key in the task requires (e.g. "train", or "train_eval"),
+        return a metadata dataframe with the following columns:
+            * relpath - How you find the file path in the original dataset.
+            * split - Split of this particular audio file.
+            * label - Label for the scene or event.
```

Comment:
Might be worth mentioning that for multilabel you add a new row for each label, so a particular audio file could show up multiple times. Not sure how we handle the case where an audio file has no label in a multilabel -- I guess cross that bridge when we get there. And then for event you should also have columns for 'start' and 'end' for each label.

Reply:
Amazing possible gotcha. https://github.com/neuralaudio/hear-preprocess/issues/21

Added the comment text you asked for